threat-model/readme.md

```diff
@@ -0,0 +1,360 @@
+2. First party context (e.g. first party cookies / information in first party storage.)
+3. User-agent provided information (e.g. user agent string, device information, user configurations.)
+4. Encrypted outputs from the proposed API.
+    1. In some cases, these are somewhat separated from the other assets (i.e. encrypted outputs may be delayed, and not connected to first party context.)
+
+
+#### 1.2.2. Capabilities
+
+1. First parties can modify client assets through the proposed APIs.
+2. First parties can choose which encrypted outputs from the proposed API are provided to aggregators.
+
+
+#### 1.2.3. Mitigations
+
+1. Modification of client assets should be limited by the API interface to only allow for intended modifications.
+2. Use of differential privacy (see [section 3. Aggregation and Anonymization](#3-Aggregation-and-Anonymization)) should be used to prevent
+
+
+### 1.3. Delegated Parties (Cross Site/App)
+
+A “delegated party” is a party which the first party embeds within their site/app, typically as a service provider to help manage the usage of the types of activities this group looks to support.
+
+Here, we assume that a delegated party is both embedded in a first party site/app, and that they are cross-site. We assume partitioned storage access/cookies (as the goal of the PATWG’s efforts are to build APIs that support functionality without the need for unpartitioned storage access/cookies.)
+
+In some cases, a first party may decide to allow a service provider to run directly in their first party context. In these cases, the service provider will be assumed, under the threat model, to have all the assets and capabilities of the first party, and not those of a delegated party.
+
+See [section 2. First Parties, Embedded Parties and Delegated Parties](#2-First-Parties-Embedded-Parties-and-Delegated-Parties) for more details on first parties vs delegated parties.
+
+
+#### 1.3.1. Assets
+
+1. All the same assets as First Parties ([section 1.2.1](#121-Assets))
+    1. Exception that 1.2.1.2 would be partitioned delegated party context.
+
+
+#### 1.3.2. Capabilities
+
+1. The same capabilities as First Parties ([section 1.2.2](#122-Capabilities))
+2. Delegated parties can modify client assets through the proposed APIs, which may include overwriting/modifying data provided by the first party or other delegated parties.
+
+
+#### 1.3.3. Mitigations
+
+1. The same mitigations as Third Parties ([section 1.2.3](#123-Mitigations))
+2. The client should prevent the ability of a delegated party to overwrite/modify data provided by the first party or other delegated parties. If this is not possible, the client should allow the first party to control and limit a delegated party's ability in this manner. (See [section 2. First Parties, Embedded Parties and Delegated Parties](#2-First-Parties-Embedded-Parties-and-Delegated-Parties) for more details.)
+
+
+### 1.4 Helper Parties and Helper Party Networks
```

Comment:
I separated out distinct parties here for "Helper parties" (as a class), "aggregators" for helper parties in an MPC, and "coordinators" for helper parties supporting a TEE. I also added Cloud providers, TEE operators, and TEE manufacturers.

@martinthomson and @marianapr, if you'd be so kind to review again, it would be much appreciated!

Reply:
Resolving this, but please feel free to open new issues and/or PRs.